### PR TITLE
[7892] committing Boutein's mynav component

### PR DESCRIPTION
**Pull request by shakirullah7892**
committed Bouteins mynav component.

### CODE REVIEW

File: angular-techDraft/src/app/my-nav/my-nav.component.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.image {
+    float: left;
+}
```
Comment (shakirullah7892) [inline angular-techDraft/src/app/my-nav/my-nav.component.css:3]: insert one line space for good coding manners.

**Review by jp-2019 — COMMENTED** [review]
agree with what shak said

File: angular-techDraft/src/app/my-nav/my-nav.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<div class="ui fluid placeholder">
+  <div class="image header">
+    <div class="container">
+      <img class="ui mini circular image" src="https://yt3.ggpht.com/-5oE3xInEpSw/AAAAAAAAAAI/AAAAAAAAAAU/5Q6ic5I5378/s88-mo-c-c0xffffffff-rj-k-no/photo.jpg">
```
Comment (shakirullah7892) [inline angular-techDraft/src/app/my-nav/my-nav.component.html:4]: Image should be a placeholder. It should not have an actual picture. See Milanote for visual comparison.

File: angular-techDraft/src/app/my-nav/my-nav.component.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    </div>
+    <div><i class="cog icon"></i></div>
+  </div>
+  <div class="paragraph">
+    <div class="line">
+      <h2>Your Profile </h2>
```
Comment (shakirullah7892) [inline angular-techDraft/src/app/my-nav/my-nav.component.html:10]: Spacing is required in between the picture placeholder and the <h2> tag of Your Profile.

**Review by shakirullah7892 — COMMENTED** [review]
Please see all changes to be made and do correct the changes. Thank you.

**Comment by shakirullah7892** [comment]
Ok so it has been agree that you do not want this component any more and that you are going to work on a whole new component. Hence, I will close this request without any further actions need on both our sides.

**Pull request closed without merging**